src/webviews/mongoClusters/documentView/documentView.tsx

```diff
@@ -170,10 +170,28 @@ export const DocumentView = (): JSX.Element => {
 
         setIsLoading(true);
 
+        let documentLength = 0;
+
         void trpcClient.mongoClusters.documentView.getDocumentById.query(documentId).then((response) => {
+            documentLength = response.length ?? 0;
+
             setContent(response);
             setIsLoading(false);
         });
+
+        trpcClient.common.reportEvent
+            .mutate({
+                eventName: 'refreshDocument',
+                properties: {
+                    ui: 'button',
+                },
+                measurements: {
+                    documentLength: documentLength,
+                },
+            })
+            .catch((error) => {
+                console.error('Failed to report event:', error);
```

Comment:
If this is error then either it should show message to user or send to telemetry. If this is debug info, please use console.debug or log.